lib/page/missing_country.rb

```diff
@@ -0,0 +1,8 @@
+module Page
+  class MissingCountry
+    attr_reader :missing
+    def initialize(country, world)
+      @missing = world[country.to_sym]
```

Comment:
It would be more in keeping with the other classes if this was `@country`, rather than `@missing`. And, after all, the thing that this variable contains is actually country, not a "missing".